vue/dynamicforms/src/components/form/inputs/input.vue

```diff
@@ -54,6 +54,16 @@ function handleNumberInput(newValue: any) {
   emits('update:modelValue', newValue);
 }
 
+const value = computed({
+  get(): any {
+    return props.modelValue;
+  },
+  set(newValue: any) {
+    handleNumberInput(newValue);
+    emits('update:modelValue', newValue);
```

Comment:
If I understand correctly... handleNumberInput already emits 'update:modelValue', therefore this is redundant.
Also do we even need handleNumberInput? We are only using it here. You could only copy the body in here and delete the function.